### PR TITLE
Foresight edition

### PR DESCRIPTION
**Pull request by hiteshnettam24**
The tutorials are bifurcated into Free and Licensed versions

### CODE REVIEW

File: Foresight-Free/hour_of_day_context.csv
Comment (venkat1m) [inline Foresight-Free/hour_of_day_context.csv:1]: Can you rename the directory to all lower case "foresight-free" and "foresight-licensed"? Is there a reason they are title cased?

**Review by hegdean — CHANGES_REQUESTED** [review]
As discussed images can be consolidated in a common folder
remove csv files from tutorial and  other items as mentioned in the call

File: Foresight-Free/trip_fare_data_sources.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+version: 2
```
Comment (vijayi1) [inline Foresight-Free/trip_fare_data_sources.yml:1]: can you create a top level trip_fare directory. we could have more tutorials, like store_sales, etc. 
so - 
foresight-free/trip_fare/....
foresight-licensed/trip_fare/...

File: Foresight-Free/trip_fare_prediction_model_2/trip_fare_2_model_train.ml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+select * from trip_fare_2_train_dataset
```
Comment (vijayi1) [inline Foresight-Free/trip_fare_prediction_model_2/trip_fare_2_model_train.ml:1]: please add a limit 2000 to the training dataset query. pycaret takes too  long.
this comment applies to tutorial-1 and 2.

File: Foresight-Licensed/trip_fare_prediction_model_1/trip_fare_1_train_dataset.ml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+select from_utc_timestamp(pickup_datetime, "EST") as pickup_datetime,
+       pickup_zipcode, dropoff_zipcode, passenger_count, fare_amount
+from {{ source('mysql','trip_table') }}
```
Comment (vijayi1) [inline Foresight-Licensed/trip_fare_prediction_model_1/trip_fare_1_train_dataset.ml:3]: please add a limit 2000 on the rows for pycaret (takes too long - please test it).
alternatively you may create two training datasets, trip_fare_1_train_dataset_small and trip_fare_1_train_dataset_large, the row limit can be added for the small dataset and used for pycaret. dl can use the large dataset (without row limit).
this comment applies to tutorials-1, 2 and 3.

**Pull request merged**